assets/data-port/import/upload/upload-page.js

```diff
@@ -40,7 +40,7 @@ export const UploadPage = ( { state, isReady, submitStartImport } ) => {
 						isPrimary
 						className="continue-button"
 						disabled={ ! isReady || isSubmitting }
-						onClick={ submitStartImport }
+						onClick={ () => submitStartImport( jobId ) }
```

Comment:
Recommendations vary a bit on this, but we could also just get the `jobId` from the state in the action creators, so we don't have to pass it around.

Reply:
Yeah, I did a bit of research before going this route and went with a few recommendations (one from Dan A. of React) saying to only do it when necessary. Maybe this is that?

Reply:
Looks like it's [done for similar things](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/store/actions.js#L307) across Gutenberg. But yeah, not critical.

Reply:
Still a bit unsure, but made the change in 8503390dc95039c17d93337d1d9b6692a691e438 c25cae4fab9530759fc2a8ad0379fac90b881ac5

Reply:
About the access to the state through the action creator, I wouldn't do that, unless we wouldn't have another option. But if it saves code, it's okay for me! I also researched a little now and there are different opinions about that.

Reply:
I've removed these commits for now in a force push.